<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="ArrayTypeMismatchException.xml" source-language="en-US" target-language="zh-CN">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5973b2b40d5674416e247fce16ebdf227be62660b.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">973b2b40d5674416e247fce16ebdf227be62660b</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-1.0,netstandard-1.1,netstandard-1.2,netstandard-1.3,netstandard-1.4,netstandard-1.5,netstandard-1.6,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.ArrayTypeMismatchException">
          <source>The exception that is thrown when an attempt is made to store an element of the wrong type within an array.</source>
          <target state="translated">当尝试在数组内存储错误类型的元素时引发的异常。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source><ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> is thrown when the system cannot convert the element to the type declared for the array.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> 系统无法将元素转换成为数组声明的类型时引发。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>For example, an element of type <ph id="ph1">&lt;xref:System.String&gt;</ph> cannot be stored in an <ph id="ph2">&lt;xref:System.Int32&gt;</ph> array because conversion between these types is not supported.</source>
          <target state="translated">例如，类型的元素<ph id="ph1">&lt;xref:System.String&gt;</ph>不能存储在<ph id="ph2">&lt;xref:System.Int32&gt;</ph>数组因为不支持这些类型之间的转换。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>It is generally unnecessary for applications to throw this exception.</source>
          <target state="translated">它是通常不必要的应用程序引发此异常。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>The following Microsoft intermediate language (MSIL) instructions throw <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> :</source>
          <target state="translated">以下 Microsoft 中间语言 (MSIL) 指令引发<ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>:</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>ldelem.<ph id="ph1">\&lt;</ph>type&gt;</source>
          <target state="translated">ldelem.<ph id="ph1">\&lt;</ph>type&gt;</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>ldelema</source>
          <target state="translated">ldelema</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>stelem.<ph id="ph1">\&lt;</ph>type&gt;</source>
          <target state="translated">stelem.<ph id="ph1">\&lt;</ph>type&gt;</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source><ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> uses the HRESULT COR_E_ARRAYTYPEMISMATCH, which has the value 0x80131503.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> 使用 HRESULT COR_E_ARRAYTYPEMISMATCH，其值 0x80131503。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>For a list of initial property values for an instance of <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>, see the <ph id="ph2">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> constructors.</source>
          <target state="translated">有关的实例的初始属性值的列表<ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>，请参阅<ph id="ph2">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>构造函数。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.ArrayTypeMismatchException">
          <source>The following code example demonstrates scenarios where <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> is thrown.</source>
          <target state="translated">下面的代码示例演示方案其中<ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>引发。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="T:System.ArrayTypeMismatchException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> class.</source>
          <target state="translated">初始化 <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> 类的新实例。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> class.</source>
          <target state="translated">初始化 <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> 类的新实例。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> property of the new instance to a system-supplied message that describes the error, such as "Source array type cannot be assigned to destination array type."</source>
          <target state="translated">此构造函数初始化<ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph>为系统提供的消息的描述错误，如"数组将源类型不能分配给目标数组类型。"的新实例的属性</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>This message takes into account the current system culture.</source>
          <target state="translated">此消息将当前系统区域性考虑在内。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>.</source>
          <target state="translated">下表显示 <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> 的实例的初始属性值。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>Property</source>
          <target state="translated">Property</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>Value</source>
          <target state="translated">“值”</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).</source>
          <target state="translated">空引用（在 Visual Basic 中为 <ph id="ph1">`Nothing`</ph>）。</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>The localized error message string.</source>
          <target state="translated">本地化的错误消息字符串。</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>The following example demonstrates the ArrayTypeMismatchException() constructor of the <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> class.</source>
          <target state="translated">下面的示例演示的 ArrayTypeMismatchException() 构造函数<ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>类。</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>It contains a function which takes two arrays as arguments and checks whether the two arrays are of the same type.</source>
          <target state="translated">它包含采用两个数组作为自变量，并检查两个数组是否属于相同类型的函数。</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor">
          <source>If the arrays are of different types, a new <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> is thrown and then caught in the calling method.</source>
          <target state="translated">如果数组在不同类型的一个新<ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>是引发，然后在调用方法中捕获。</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>A <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> that describes the error.</source>
          <target state="translated">描述该错误的 <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> class with a specified error message.</source>
          <target state="translated">用指定的错误消息初始化 <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> 类的新实例。</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>The content of the <ph id="ph1">`message`</ph> parameter is intended to be understood by humans.</source>
          <target state="translated"><ph id="ph1">`message`</ph> 参数的内容应为人所理解。</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">此构造函数的调用方需要确保此字符串已针对当前系统区域性进行了本地化。</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>.</source>
          <target state="translated">下表显示 <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> 的实例的初始属性值。</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>Property</source>
          <target state="translated">Property</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>Value</source>
          <target state="translated">“值”</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).</source>
          <target state="translated">空引用（在 Visual Basic 中为 <ph id="ph1">`Nothing`</ph>）。</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>The error message string.</source>
          <target state="translated">错误消息字符串。</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>The following example demonstrates the ArrayTypeMismatchException(String) constructor of the <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> class.</source>
          <target state="translated">下面的示例演示的 ArrayTypeMismatchException(String) 构造函数<ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>类。</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>It contains a function which takes two arrays as arguments and checks whether the two arrays are of the same type.</source>
          <target state="translated">它包含采用两个数组作为自变量，并检查两个数组是否属于相同类型的函数。</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String)">
          <source>If the arrays are of different types, a new <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> is thrown and then caught in the calling method.</source>
          <target state="translated">如果数组在不同类型的一个新<ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>是引发，然后在调用方法中捕获。</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The object that holds the serialized object data.</source>
          <target state="translated">承载序列化对象数据的对象。</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The contextual information about the source or destination.</source>
          <target state="translated">关于来源和目标的上下文信息</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> class with serialized data.</source>
          <target state="translated">用序列化数据初始化 <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> 类的新实例。</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</source>
          <target state="translated">此构造函数在反序列化过程中调用，以便重建通过流传输的异常对象。</target>       </trans-unit>
        <trans-unit id="140" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>For more information, see <bpt id="p1">[</bpt>XML and SOAP Serialization<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>.</source>
          <target state="translated">有关详细信息，请参阅<bpt id="p1">[</bpt>XML 和 SOAP 序列化<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>。</target>       </trans-unit>
        <trans-unit id="141" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">解释异常原因的错误消息。</target>       </trans-unit>
        <trans-unit id="142" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">导致当前异常的异常。</target>       </trans-unit>
        <trans-unit id="143" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> parameter is not a null reference, the current exception is raised in a <ph id="ph1">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated">如果 <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> 参数不为空引用，则在处理内部异常的 <ph id="ph1">&lt;see langword="catch" /&gt;</ph> 块中引发当前异常。</target>       </trans-unit>
        <trans-unit id="144" translate="yes" xml:space="preserve" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">使用指定的错误消息和对作为此异常原因的内部异常的引用来初始化 <ph id="ph1">&lt;see cref="T:System.ArrayTypeMismatchException" /&gt;</ph> 类的新实例。</target>       </trans-unit>
        <trans-unit id="145" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">因前一个异常而直接引发的异常应在 <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> 属性中包含对前一个异常的引用。</target>       </trans-unit>
        <trans-unit id="146" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor, or a null reference (<ph id="ph2">`Nothing`</ph> in Visual Basic) if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> 属性返回传递到构造函数中的相同值，或者如果 <ph id="ph2">`Nothing`</ph> 属性不向构造函数提供内部异常值，则为空引用（在 Visual Basic 中为 <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph>）。</target>       </trans-unit>
        <trans-unit id="147" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>.</source>
          <target state="translated">下表显示 <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> 的实例的初始属性值。</target>       </trans-unit>
        <trans-unit id="148" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>Property</source>
          <target state="translated">Property</target>       </trans-unit>
        <trans-unit id="149" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>Value</source>
          <target state="translated">“值”</target>       </trans-unit>
        <trans-unit id="150" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The inner exception reference.</source>
          <target state="translated">内部异常引用。</target>       </trans-unit>
        <trans-unit id="151" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The error message string.</source>
          <target state="translated">错误消息字符串。</target>       </trans-unit>
        <trans-unit id="152" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> class.</source>
          <target state="translated">下面的代码示例演示<ph id="ph1">&lt;xref:System.ArrayTypeMismatchException.%23ctor%2A&gt;</ph>构造函数的<ph id="ph2">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>类。</target>       </trans-unit>
        <trans-unit id="153" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>It contains a function that takes two arrays as arguments and checks whether the two arrays are of the same type.</source>
          <target state="translated">它包含采用两个数组作为自变量并检查两个数组是否属于同一类型的函数。</target>       </trans-unit>
        <trans-unit id="154" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.ArrayTypeMismatchException.#ctor(System.String,System.Exception)">
          <source>If the arrays are of different types, a new <ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph> is thrown and then caught in the calling method.</source>
          <target state="translated">如果数组在不同类型的一个新<ph id="ph1">&lt;xref:System.ArrayTypeMismatchException&gt;</ph>是引发，然后在调用方法中捕获。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>